src/test/java/duke/command/DeadlineCommandTest.java

```diff
@@ -0,0 +1,56 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.StorageStub;
+import duke.TaskList;
+import duke.Ui;
+
+import duke.task.DeadlineTask;
+import duke.task.TodoTask;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DeadlineCommandTest {
+
+    @Test
+    public void run_wronglyFormattedDeadlineCommand1_exceptionThrown() {
+        DeadlineCommand dlCmd = new DeadlineCommand("deadline wronginput", new Ui());
+        assertThrows(DukeException.class, () -> {
+            dlCmd.run(new StorageStub(), new TaskList());
+        });
+    }
+
+    @Test
+    public void run_wronglyFormattedDeadlineCommand2_exceptionThrown() {
+        DeadlineCommand dlCmd = new DeadlineCommand("deadline testWronglyFormatted DeadlineCmd /at 2020-01-01 1800", new Ui());
+        assertThrows(DukeException.class, () -> {
+            dlCmd.run(new StorageStub(), new TaskList());
+        });
+    }
```

Comment:
Your code is well written and understandable. Your testing is also rigorous and complete. Keep up the good work!